docs/source/guides/network_node/staking_guide.rst

```diff
@@ -247,15 +247,23 @@ Once you have created one or more stakes, you can view all active stake for conn
 
     (nucypher)$ nucypher stake list
 
-    ======================================= Active Stakes =========================================
-
-    | ~ | Staker | Worker | # | Value    | Duration     | Enactment
-    |   | ------ | ------ | - | -------- | ------------ | -----------------------------------------
-    | 0 | 0xbb01 | 0xdead | 0 | 15000 NU | 41 periods . | Aug 04 12:15:16 CEST - Sep 13 12:15:16 CEST
-    | 1 | 0xbb02 | 0xbeef | 1 | 15000 NU | 30 periods . | Aug 20 12:15:16 CEST - Sep 18 12:15:16 CEST
-    | 2 | 0xbb03 | 0x0000 | 0 | 30000 NU | 30 periods . | Aug 09 12:15:16 CEST - Sep 9 12:15:16 CEST
-
-If the Worker in the list is shown as ``0x0000``, it means that you haven't yet
+    Network <NETWORK_NAME> ═══════════════════════════════
+    Staker 0x270b3f8af5ba2B79ea3Bd6a6Efc7ecAB056d3E3f ════
+    Worker NO_WORKER_BONDED ════
+    --------------  -----------------------------------
+    Status          Never Made a Commitment (New Stake)
+    Restaking       Yes (Unlocked)
+    Winding Down    No
+    Unclaimed Fees  0 ETH
+    Min fee rate    0 ETH
+    --------------  -----------------------------------
+    ╒═══════╤══════════╤═════════════╤═════════════╤═══════════════╕
+    │   Idx │ Value    │   Remaining │ Enactment   │ Termination   │
+    ╞═══════╪══════════╪═════════════╪═════════════╪═══════════════╡
+    │ 	0   │ 30000 NU │      	  31 │ Jun 19 2020 │ Jul 19 2020   │
```

Comment:
Odd formatting here

Reply:
Seems to be weird handling of rst by github - no problem when viewed here: https://github.com/nucypher/nucypher/blob/4d3cbe27990e92b26d07716f2adbeeef9ecdb0b8/docs/source/guides/network_node/staking_guide.rst